### PR TITLE
Completed Express POST JSON

### PR DESCRIPTION
**Pull request by PaulKillian**
image
image
image
image

### CODE REVIEW

File: express-post-json/index.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+app.use(express.json());
+
+app.get('/api/grades', (req, res) => {
+  res.json(grades);
+});
+
+app.post('/api/grades', (req, res) => {
+  res.status(201);
+  req.body['id:'] = `${nextId}`;
```
Comment (thebearingedge) [inline express-post-json/index.js:15]: you can just use `.id` and leave the `nextId` as a number

File: express-post-json/index.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+  grades.push(req.body);
+  nextId++;
+  res.json(grades);
```
Comment (thebearingedge) [inline express-post-json/index.js:18]: The server should respond with the added grade and not the whole list.

**Review by thebearingedge — CHANGES_REQUESTED** [review]

File: express-post-json/index.js (hunk 1 above)
Comment (PaulKillian) [reply express-post-json/index.js]: Got it! Updated

File: express-post-json/index.js (hunk 2 above)
Comment (PaulKillian) [reply express-post-json/index.js:18]: Oh, missed this comment. Will fix.

**Review by thebearingedge — APPROVED** [review]

**Pull request merged**